app/services/idv/steps/in_person/address_step.rb

```diff
@@ -20,18 +20,38 @@ def call
 
         def extra_view_variables
           {
-            pii: flow_session[:pii_from_user],
-            updating_address: flow_session[:pii_from_user].has_key?(:address1),
+            form:,
+            pii:,
+            updating_address:,
           }
         end
 
         private
 
-        def form_submit
-          Idv::InPerson::AddressForm.new.submit(
-            permit(*Idv::InPerson::AddressForm::ATTRIBUTES),
+        def updating_address
+          flow_session[:pii_from_user].has_key?(:address1)
+        end
+
+        def pii
+          data = flow_session[:pii_from_user]
+          data = data.merge(flow_params) if params.has_key?(:in_person_address)
+          data.deep_symbolize_keys
+        end
+
+        def flow_params
+          params.require(:in_person_address).permit(
+            # params.permit(
+            *Idv::InPerson::AddressForm::ATTRIBUTES,
           )
```

Comment:
I think we can remove the comment
```suggestion
          params.require(:in_person_address).permit(
            *Idv::InPerson::AddressForm::ATTRIBUTES,
          )
```